ExtLibs/Mavlink/message_definitions/ardupilotmega.xml

```diff
@@ -1390,5 +1390,12 @@
       <field name="AOA" type="float">Angle of Attack (degrees)</field>
       <field name="SSA" type="float">Side Slip Angle (degrees)</field>
     </message>
+    <!-- Angle of Attack and Side Slip Angle message -->
```

Comment:
I would normally grab this from upstream this ensures whatever is in upstream mp supports with the approx. checking time